src/include/miopen/readonlyramdb.hpp

```diff
@@ -36,6 +36,21 @@
 
 namespace miopen {
 
+extern bool& testing_rordb_embed_fs_override();
```

Comment:
Please move this function into `miopen::debug` namespace. Why: It is required only for testing.